geoips/image_utils/maps.py

```diff
@@ -167,9 +167,14 @@ def meridians(area_def, grid_size):
 
     corners = area_def.corners
     lons = [np.rad2deg(corn.lon) for corn in corners]
-    llcrnrlon = lons[3]
-    urcrnrlon = lons[1]
-
+    if area_def.proj_dict["lat_0"] > 0:
+        crn_idx = 0
+        mc_idx = 1
+    else:
+        crn_idx = 3
+        mc_idx = 2
+    llcrnrlon = lons[crn_idx]
+    urcrnrlon = lons[mc_idx]
```

Comment:
Why is this needed? Can we add a comment here describing the purpose?

Reply:
Not sure,  will look into it

Reply:
I created an issue to look into this. I don't believe that this is quite correct. It would likely fail for any negative lat_0 other than -90.